### ISSUE TITLE
Overlay imaginary tiles in 3d points

### ISSUE THREAD

**Issue by oscarmartinezrubi** [NLeSC/ahn-pointcloud-viewer#15]
See http://potree.org/demo/experimental/potree_map_projections/examples/viewer_proj.html
or 
http://iowalidar.com/

**Comment by Maartenvm** [comment]
_Imaginary_ tiles? done!

**Comment by Maartenvm** [comment]
Srsly though, imagery tiles would be cool :P

**Comment by oscarmartinezrubi** [comment]
yes sure...anyway, i renamed to imagery...

**Comment by m-schuetz** [comment]
My experience with this:
I tried using open layers to fetch a 128x128 or 256x256 image for each octree node and then project the image onto the points in a node. Unfortunately, the performance of this approach was not in an acceptable range. Since it was rendering into a canvas element or at least using DOM elements it also couldn't be moved into a parallel WebWorker.

My next try would have been to directly load image tiles from within an WebWorker which also brings it into the correct format so the main thread can pass it to the GPU without further processing.

**Comment by Maartenvm** [comment]
Yes, I've come to the same conclusion. We can get the tiles for the (official) aerial photography of the Netherlands from the Dutch cadastre institution but I'm afraid I just can't spare the time to implement this.
I'm therefore filing this under "Future work".
